script/Ciel/Day 14/14_01B_03_CIEL14B_4_1E.txt

```diff
@@ -286,27 +348,32 @@ That woman―――She said it herself that she was 'tired of it', so what the h
 [sha:0ce0e5f93f7e054969fe3835a7c375da42d86739]{
 -- Page 127, Offset 40618.
 -- だが暴虐はそれだけに留まらなかった。
-But her outrage didn't just stop there.
+But her tyranny doesn't end there.
 }
 [sha:9f8ea3f9adb0ed14143e21010c65c30cc2f14abb]{
 -- Page 127, Offset 40619.
 -- アイツはシエル先輩を胸元まで浮かせると、さも不愉快そうな仕草で……頬を膨らませたように見えなくもなかった……先輩を指で弾いた。
-She floated Ciel-senpai up to her chest, and with a extremely unpleased gesture... as if she had puffed out her cheeks like never before... she flicked Senpai away with her fingers.
+// yeet
+After raising Ciel-senpai back up to chest height,
+ and with an extremely displeased gesture――
+it's hard to tell, but I think she was pouting――
+flicks Ciel-senpai away with her finger.
 }
 [sha:5b772fc94e2309ca26d78901f3d07cbff89f093c]{
 -- Page 127, Offset 40620.
 -- コオン、と硬い物を叩くような震動が響く。
-The sound of a hard object being hit echoes.
+I hear the echoing sound of a hard object being struck.
 }
 [sha:8a2078647aec87e9026771d01098f98d81be34a4]{
 -- Page 128, Offset 40621.
 -- アルクェイドの指で弾かれた先輩。
-Senpai is flicked away by Arcueid's fingers.
+Arcueid sent Senpai flying with that flick.
 }
 [sha:be2f3d89c47a3346b565a6b78e9510bc9a1b3cff]{
 -- Page 128, Offset 40622.
 -- それは一条の矢のように夜空を横切り、『事象収納』の外側―――俺たちの学校の校舎に着弾した。
-She crosses the night sky like a single arrow, crashing outside the 'Event Storage' at our school.
+She streaks across the night sky like an arrow,
+ before crashing back down to earth at our school――outside the range of the 'Event Storage'.
```

Comment:
The first sentence loses its purpose like this
Let's combine these and spread out the clauses
>Senpai is sent streaking across the night sky like an arrow in flight by Arcueid's flick.

>When she lands, she has arrived all the way at the edge of the Event Storage――at our school.